Backend/lib/payment/src/Lib/Storage/Tabular/Payment/PaymentOrder.hs

```diff
@@ -18,26 +18,24 @@
 {-# LANGUAGE TemplateHaskell #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
-module Storage.Tabular.Payment.PaymentOrder where
+module Lib.Storage.Tabular.Payment.PaymentOrder where
```

Comment:
Why this `Payment` lib is basically nameless? You placed all `Payment` specific stuff in common for all libls `Lib` space